Simplenote/Options.swift

```diff
@@ -68,6 +68,7 @@ extension Options {
         }
         set {
             defaults.set(newValue.rawValue, forKey: .listSortMode)
+            UserDefaults(suiteName: SimplenoteConstants.sharedGroupDomain)?.set(newValue.rawValue, forKey: .listSortMode)
```

Comment:
so if the Defaults aren't initialized with `suiteName`, the value isn't accessible?

Problem with this approach is: unless you set the `listSortMode` again, then, it won't be accessible by the extension.

Reply:
Good point.  I added a method to sync the setting in `applicationDidEnterBackground` as well.  That should solve the issue of the setting not syncing to the widgets/intents until it is updated